Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -869,7 +869,7 @@ WarpXParticleContainer::PushX (int lev, amrex::Real dt)
 
     amrex::LayoutData<amrex::Real>* cost = WarpX::getCosts(lev);
 
-#ifdef _OPENMP
+#ifdef AMREX_USE_OMP
 #pragma omp parallel
```

Comment:
Missing `if (amrex::Gpu::notInLaunchRegion())`?

Reply:
I agree with you that `if (amrex::Gpu::notInLaunchRegion())` should be there

Reply:
```suggestion
#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
```